src/components/ArticleCard.jsx

```diff
@@ -1,20 +1,26 @@
 import { useEffect, useState } from "react";
-import { fetchArticles } from "../utils/api";
+import { fetchArticleById } from "../utils/api";
+import { useParams } from "react-router-dom";
 
 const ArticleCard = () => {
   const [articleCard, setArticleCard] = useState([]);
+  const { article_id } = useParams();
 
   useEffect(() => {
-    fetchArticles(1).then((response) => {
-      console.log(response.data.article);
+    fetchArticleById(article_id).then((response) => {
```

Comment:
I recommend you to set up loading state and display a message to the user while the article is loading.